src/App.js

```diff
@@ -1,11 +1,12 @@
+/* eslint-disable no-unused-vars */
+import Calculator from './components/Calculator.js';
+
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <p>
-          Math Magicians build version 01 test
-        </p>
-      </header>
+      <div>
+        <Calculator />
+      </div>
     </div>
   );
 }
```

Comment:
- [ ] The `App.js` component should be class-based as per the project's requirements even though functional component is very correct. Please change this functional component to a class component. :+1:
- [ ] You also need to import `React, {Component}` when you convert the functional component to class component extended from `React Component`.